install_sct.bat

```diff
@@ -58,6 +58,11 @@ echo:
 echo ### Downloading model files and binaries...
 FOR %%D IN (PAM50 gm_model optic_models pmj_models deepseg_sc_models deepseg_gm_models deepseg_lesion_models c2c3_disc_models binaries_win) DO sct_download_data -d %%D -k || goto error
 
+rem Copying SCT scripts to an isolated folder (so we can add scripts to the PATH without adding the entire venv_sct)
+echo:
+echo ### Copying SCT's CLI scripts to %CD%\bin\
+xcopy %CD%\venv_sct\Scripts\*sct*.* %CD%\bin\ /v /y /q /i

```

Comment:
Oh wow, `xcopy`! I hadn't seen that command in ages :laughing:
Is this missing `|| goto error`, though?

Reply:
Good catch! 😅